OWNERS

```diff
@@ -0,0 +1,15 @@
+# See Config struct in https://github.com/kubernetes/test-infra/blob/master/prow/repoowners/repoowners.go
+
+# Reviewers may be auto-assigned by blunderbuss
+reviewers:
```

Comment:
When the reviewers and approvers section is the same, you can just have the approvers section (we'll read reviewers from the approvers section in this scenario)